src/Tickets/Admin/Tickets/List_Table.php

```diff
@@ -377,7 +390,18 @@ public function column_name( $item ): string {
 			'ticket_link' => $edit_post_link,
 		];
 
-		return $template->template( 'tickets/column/name', $context, false );
+		$name = $template->template( 'tickets/column/name', $context, false );
+
+		/**
+		 * Filters the name for the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $name The name for the All Tickets Table.
+		 *
+		 * @return string
+		 */
+		return apply_filters( 'tec_tickets_all_tickets_table_column_name', $name, $item );
```

Comment:
We'd typically do two of these, the second one would have the name in the filter, i.e.
`tec_tickets_all_tickets_table_column_name_{$name}`